charts/kubescape-operator/values.yaml

```diff
@@ -772,6 +772,12 @@ prometheusExporter:
     targetPort: 8080
     protocol: TCP
 
+  # Additional volumes to be mounted on the vulnerability scanning microservice
```

Comment:
If we are fixing a copy-paste mistake let's also fix the comments :)

Reply:
Good catch! Will do accordingly.

Reply:
Comments have been updated ;-)